src/client_side_request.h

```diff
@@ -76,6 +76,9 @@ class ClientHttpRequest
     /// the request. To set the virgin request, use initRequest().
     void resetRequest(HttpRequest *);
 
+    /// the processing tags associated with this request transaction.
+    LogTags &logType() { return al->cache.code; }
```

Comment:
To facilitate future const-correctness, please also add (and use) a `const` method returning a `const` reference.

I also do not see a good reason to keep the mismatching "Type" suffix for the new method(s). We return LogTags, not LogType. I suggest calling the const method loggingTags().

The situation with the non-const method is a bit trickier: The vast majority of writers call the update method, and there are lots of them! Encapsulating that frequently reused logic into updateLoggingTags() method could be a good idea. The remaining writers, if any, can use raw ALE. With that design, there will be no non-constant loggingTags() method.